### DISCUSSION TITLE
can you add a config setting to disable the extra insanity sound clips?

### DISCUSSION THREAD

**Discussion by dannybagboy** [BudgetAirpods/InsanityRemastered#21] [Ideas]
Me & my friends find the sound effects used tend to break the immersion more than enhance it, but we do enjoy the other changes youve made to the insanity system. Can you add an option to disable just the extra sound effects, and leave it as vanilla in the config options?

**Comment by BudgetAirpods** [comment]
Sure, I'm assuming you mean keep the vanilla sound effects as hallucinations but remove the other ones? If so, I can do that.

**Comment by BudgetAirpods** [comment]
Okay. Just added this. Not released yet, of course.
